dandi/publish/tests/test_stats.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_stats_none(api_client):
+    resp = api_client.get('/api/stats/')
+    assert {
+        'draft_count': 0,
+        'published_count': 0,
+        'user_count': 0,
+        'species_count': 0,
+        'subject_count': 0,
+        'cell_count': 0,
+        'size': 0,
+    } == resp.json()
```

Comment:
```suggestion
    assert api_client.get('/api/stats/').data == {
        'draft_count': 0,
        'published_count': 0,
        'user_count': 0,
        'species_count': 0,
        'subject_count': 0,
        'cell_count': 0,
        'size': 0,
    }
```

The other tests which call the API use this pattern. By accessing `.data`, you force DRF to apply the serializer and convert things to primitive Python objects, but don't need to invoke the transformation to / from a string (which is outside the scope of application testing), which makes the tests faster.

If you approve, you'll need to update the tests below too.

Reply:
The attribute `data` is not defined on `JsonResponse` objects. The other endpoints are using model instances, which do supply the `data` attribute. This endpoint is returning a JSON object directly, which I think is appropriate here.

Reply:
I now see that DRF has it's own `Response` class, I will implement this.